starboard/BUILD.gn

```diff
@@ -61,25 +66,26 @@ group("gn_all") {
   if (build_with_separate_cobalt_toolchain && use_contrib_cast) {
     deps += [ "//starboard/contrib/cast/cast_starboard_api/samples:cast" ]
   }
-
-  if (!sb_is_evergreen) {
-    if (sb_is_evergreen_compatible) {
-      deps += [
-        "//starboard/elf_loader:elf_loader_test_install($starboard_toolchain)",
-        "//starboard/loader_app:installation_manager_test_install($starboard_toolchain)",
-        "//starboard/loader_app:reset_evergreen_update_test_install($starboard_toolchain)",
-      ]
-
-      if (current_toolchain == starboard_toolchain) {
+  if (enable_evergreen_code) {
+    if (!sb_is_evergreen) {
+      if (sb_is_evergreen_compatible) {
         deps += [
-          "//third_party/crashpad/crashpad/client",
-          "//third_party/lz4_lib:lz4",
-        ]
-        data_deps = [
-          "//starboard/loader_app($starboard_toolchain)",
-          "//third_party/crashpad/crashpad/handler:crashpad_handler(//$starboard_path/toolchain:native_target)",
-          "//third_party/crashpad/crashpad/tools:crashpad_database_util(//$starboard_path/toolchain:native_target)",
+          "//starboard/elf_loader:elf_loader_test_install($starboard_toolchain)",
+          "//starboard/loader_app:installation_manager_test_install($starboard_toolchain)",
+          "//starboard/loader_app:reset_evergreen_update_test_install($starboard_toolchain)",
         ]
+        if (current_toolchain == starboard_toolchain) {
+          deps += [ "//third_party/lz4_lib:lz4" ]
+          if (enable_evergreen_code) {
+            deps += [ "//third_party/crashpad/crashpad/client" ]
+
+            data_deps = [
+              "//starboard/loader_app($starboard_toolchain)",
+              "//third_party/crashpad/crashpad/handler:crashpad_handler(//$starboard_path/toolchain:native_target)",
+              "//third_party/crashpad/crashpad/tools:crashpad_database_util(//$starboard_path/toolchain:native_target)",
+            ]
+          }
+        }
       }
     }
   } else {  # !sb_is_evergreen
```

Comment:
This else statement should either be in the scope of the enable_evergreen_code conditional or that conditional should be moved a level deeper

Reply:
pushed it inside, a bunch of targets in that section dont build rn